suite-native/biometrics/src/useBiometrics.tsx

```diff
@@ -19,19 +21,18 @@ export const useBiometrics = () => {
     const { isBiometricsOptionEnabled } = useIsBiometricsEnabled();
     const appState = useRef(AppState.currentState);
     const { isUserAuthenticated, setIsUserAuthenticated } = useIsUserAuthenticated();
+    const goneToBackgroundAt = useRef(Number.MAX_SAFE_INTEGER);
```

Comment:
I would prefer to set the default value to `null` rather than `Number.MAX_SAFE_INTEGER`
```suggestion
    const goneToBackgroundAt = useRef(null);
```

Then you could change line `28` as follows:
```jsx
if (!goneToBackgroundAt.current || goneToBackgroundAt.current > Date.now() - HALF_MINUTE) {
```

IMO the `Number.MAX_SAFE_INTEGER` value is counterintuitive and does not bring any value to this code. But maybe it is only my preference, won't block this PR by this.